src/main/java/core/basesyntax/figures/IFigure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.figures;
+
+public interface IFigure {

```

Comment:
I would recommend to be more precise with interface name here. For example, `AreaCalculator` name will be more descriptive. Also, when renaming interface think about method `getArea()`. I would say, it is more suitable here to use `calculateArea()`, as indeed we have some computation, rather than fetching predefined data.